keyboards/cxt_studio/12e3/keymaps/vial/config.h

```diff
@@ -6,53 +6,7 @@
 #define VIAL_UNLOCK_COMBO_ROWS {0, 0}
 #define VIAL_UNLOCK_COMBO_COLS {2, 3}
 
```

Comment:
```suggestion

#define NO_ACTION_ONESHOT
```
saves another 1,208 bytes